main/.vitepress/components/Zoe-Version.vue

```diff
@@ -0,0 +1,12 @@
+<script setup>
+import { useData } from 'vitepress'
+const { theme } = useData()
+const { zoeVersion, zoeDocsUpdated } = theme.value
+</script>
+
+<template>
+<div class="zoe-version">
+    Zoe {{ zoeVersion }}. Last updated
+    {{ zoeDocsUpdated }}.
```

Comment:
For the future, do we we get value out of this? It's hardcoded to `August 25, 2022`.  VuePress had a `lastUpdatedAt` plugin for individual `.md`'s that is also available in VitePress (and active on our site)

Reply:
I think it's supposed to indicate when the zoe code was last updated. It might be nice to say what version of code we're documenting.

But yeah... currently, this is just noise.